include/bluetooth/conn.h

```diff
@@ -698,8 +733,9 @@ struct bt_conn_auth_cb {
 	/** @brief notify that pairing process has failed.
 	 *
 	 * @param conn Connection object.
+	 * @param reason Pairing failed reason
 	 */
-	void (*pairing_failed)(struct bt_conn *conn);
+	void (*pairing_failed)(struct bt_conn *conn, u8_t reason);
```

Comment:
Same here. Using the enum name will make it clear where to look up the possible values.

Reply:
switched to enum.